doc/getting_started/getting_started_go.rst

```diff
@@ -245,3 +245,197 @@ To send bare Lua code for execution, use ``Eval``:
 .. code-block:: go
 
     resp, err = connection.Eval("return 4 + 5", []interface{}{})
+
+.. _getting_started-go-comparison:
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Feature comparison
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are two more connectors from open-source community available:
+`viciious/go-tarantool <https://github.com/viciious/go-tarantool>`_ and
+`FZambia/tarantool <https://github.com/FZambia/tarantool>`_. 
+
+Last update: December 2021
+
+..  list-table::
+    :header-rows: 1
+    :stub-columns: 1
+
+    *   -
+        -   `tarantool/go-tarantool <https://github.com/tarantool/go-tarantool>`_
+        -   `viciious/go-tarantool <https://github.com/viciious/go-tarantool>`_
+        -   `FZambia/tarantool <https://github.com/FZambia/tarantool>`_
+
+    *   -   License
+        -   BSD 2-Clause
+        -   MIT
+        -   BSD 2-Clause
+
+    *   -   Last update
+        -   2021
```

Comment:
I'd avoid using numbers like this in doc - since they will become obsolete soon and must be updated periodically. This also related to Github Stars - those are already wrong.

Reply:
I would leave some numbers even if they'll not be ideal at any point of time. It gives some understanding about popularity and whether the project is maintained. It is likely what a user is interested in.

Reply:
Ok  - got it!